src/main/java/gil/server/controllers/StaticFileHandler.java

```diff
@@ -9,14 +9,23 @@
     public static BiFunction<Request, Response, Response> get =
             (request, response) -> {
                 String uri = request.getURI();
+                byte[] body = "".getBytes();
 
                 try {
-                    String body = StaticFileUtils.getFileContent(uri);
+                    switch(StaticFileUtils.getFileType(uri)) {
+                        case "TEXT":
+                            body = StaticFileUtils.getTextFileContent(uri);
+                            response.addHeader(HTTPProtocol.CONTENT_TYPE,"text/html; charset=UTF-8");
+                            break;
+                        case "JPEG":
+                            body = StaticFileUtils.getImageFileContent(uri);
```

Comment:
if the body is being set in the switch, why not make the header content type var a variable too and set the response with it below with everyone else

Reply:
Very nice. I've refactored the handler to implement this.